docs/iris/example_code/General/polynomial_fit.py

```diff
@@ -0,0 +1,51 @@
+"""
+Fitting a polynomial
+====================
+
+This example demonstrates adding a polynomial fit to a 1D plot of data from
+an Iris cube.
+
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+import iris
+import iris.plot as iplt
+import iris.quickplot as qplt
+
+
+def main():
+    fname = iris.sample_data_path('A1B_north_america.nc')
+    cube = iris.load_cube(fname)
+
+    # Extract a single time cross-section at a latitude and longitude point.
```

Comment:
I wouldn't describe this as a cross section, that is usually reserved for a straight-line through a 2D space (i.e. has units of space rather than time). It should be referred to as a time-series.